OmiseSwift/API Models/Capability.swift

```diff
@@ -86,7 +86,7 @@ extension Capability {
     public struct Method: Codable, Equatable {
         public let payment: Payment
         public let supportedCurrencies: Set<Currency>
-        
+        public let banks: [BankStatus]
```

Comment:
"Status" seems too specific. "Details" might better capture the nature of the information.